modin/engines/base/frame/partition_manager.py

```diff
@@ -511,21 +595,30 @@ def simple_shuffle(cls, axis, partitions, map_func, lengths):
 
     @classmethod
     def concat(cls, axis, left_parts, right_parts):
-        """Concatenate the blocks with another set of blocks.
-
-        Note: Assumes that the blocks are already the same shape on the
-            dimension being concatenated. A ValueError will be thrown if this
-            condition is not met.
+        """
+        Concatenate the blocks of partitions with another set of blocks.
 
-        Args:
-            axis: The axis to concatenate to.
-            right_parts: the other blocks to be concatenated. This is a
-                BaseFrameManager object.
+        Parameters
+        ----------
+        axis : int
+            The axis to concatenate to.
+        left_parts : np.ndarray
+            NumPy array of partitions to concatenate with.
+        right_parts : np.ndarray or list
+            NumPy array of partitions to be concatenated.
 
         Returns
         -------
-            A new BaseFrameManager object, the type of object that called this.
+        np.ndarray
+            A new NumPy array with concatenated partitions.
+
+        Notes
+        -----
+        Assumes that the blocks are already the same shape on the
+        dimension being concatenated. A ValueError will be thrown if this
+        condition is not met.
         """
+        # TODO: isinstance
```

Comment:
could the author of this clarify this `TODO`? @YarShev @prutskov

Reply:
comment was expanded

Reply:
Probably, @prutskov